pkg/util/common.go

```diff
@@ -44,11 +39,5 @@ func (t Tree) String() string {
 
 // IsFilePath reports whether the given URL is a local file path.
 func IsFilePath(path string) bool {
```

Comment:
Can you help to clarify? When we call helm.NewHelmRenderer, we pass in installation package path, which is a HTTP URL, then in NewHelmRender we determine to return render base on compiled in, file or HTTP. But there we have only two render, one is for local file one is for complied in. I assume we are going to integrate with the URLFetcher change there. If that's the case, the check here is not enough to make sure it must be a local file path, right?

Reply:
Yes, that's correct. We have to use priority ordering at the caller with URL first, since there's really no pattern we can look for to distinguish a file path. One could even argue that what we have here is not strictly correct because you could have a file path that doesn't match.